tools/LibraryVersionsGenerator/PackageVersionDefinitions.cs

```diff
@@ -28,6 +28,8 @@ internal static class PackageVersionDefinitions
             Versions = new List<string>
             {
                 "8.0.0",
+                "8.1.1",
+                "8.1.2",
```

Comment:
My only question here is, which versions and how many we should pick for testing.
Testing all patches seems not reasonable.

Reply:
Race condition 😉  See https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2742#discussion_r1262224703